recirq/qcqmc/hamiltonian.py

```diff
@@ -0,0 +1,319 @@
+from pathlib import Path
+from typing import Tuple, Union
+
+import attrs
+import fqe
+import h5py
+import numpy as np
+import openfermion as of
+from fqe.hamiltonians.restricted_hamiltonian import RestrictedHamiltonian
+from fqe.openfermion_utils import integrals_to_fqe_restricted
+from openfermionpyscf._run_pyscf import compute_integrals  # type: ignore
+from pyscf import ao2mo, fci, gto, scf
+
+from recirq.qcqmc.config import OUTDIRS
+from recirq.qcqmc.data import Data, Params, get_integrals_path
+
+
+@attrs.frozen
+class LoadFromFileHamiltonianParams(Params):
+    """Class for storing the parameters that specify loading integrals from a file."""
+
+    name: str
+    integral_key: str
+    n_orb: int
+    n_elec: int
+    do_eri_restore: bool = False
+    path_prefix: str = ""
+
+    def _json_dict_(self):
+        return attrs.asdict(self)
+
+    @property
+    def path_string(self, prefix: str = "") -> str:
+        return self.path_prefix + OUTDIRS.DEFAULT_HAMILTONIAN_DIRECTORY + self.name
+
+
+# xyz coordinates of an atom
+PositionT = Tuple[float, float, float]
+# [atom name, xyz]
+GeomT = Tuple[str, PositionT]
```

Comment:
nit: I would put constants at the top above the class.

Reply:
Done.